src/main/java/org/jenkinsci/test/acceptance/po/JobsMixIn.java

```diff
@@ -35,7 +35,11 @@ public JobsMixIn(ContainerPageObject context) {
             }
         }).click();
 
-        clickButton("OK");
+        //Sometimes we need to give some time to ok button to become enabled
+        waitTillEnabledAndClick("OK", 10, TimeUnit.SECONDS);
+
+        //sometimes job creation is not fast enough, so make sure it's finished before continue
+        waitFor(by.name("config"));
```

Comment:
Is this waiting for config form to be interaction ready? In case it is, I prefer to do so in `Job.configure()` if really needed.